### PR TITLE
Feature/28/amir lock pop

### PR DESCRIPTION
**Pull request by Amirjab21**
Page that allows them to connect with wallet, and lock pop for a certain amount of time. The only thing not functional on this page is the button that lets them buy more POP - wasn't sure which contract to use for this.

READ HERE.

You must run the script deployWithValues.js in order to deploy the staking and mock contract. You must then grab the addresses of these contracts from the console and put them into the corresponding const's in lockPop.tsx (line ~45)

Also make sure your metamask wallet is connecting to localhost, not ethereum mainnet.

### CODE REVIEW

**Comment by anthonymartin** [comment]
@Amirjab21 can you resolve the conflicts in the PR?

**Comment by Amirjab21** [comment]
> @Amirjab21 can you resolve the conflicts in the PR?

done

File: packages/app/styles/globals.css [hunk 1]
```diff
@@ -46,3 +49,147 @@ a {
     @apply bg-primaryLight shadow-lg transform scale-102 text-white font-bold text-lg;
   }
 }
+
+
+.lock-pop-title {
```
Comment (anthonymartin) [inline packages/app/styles/globals.css:54]: this project includes styled components and the tailwind css library. in the case of custom css, we should resort to using it with styled components, and only if tailwind css does not have the design elements required

File: packages/contracts/scripts/deployWithValues.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// import { parseEther } from "@ethersproject/units";
```
Comment (anthonymartin) [inline packages/contracts/scripts/deployWithValues.js:1]: can unused comments be removed?

**Review by anthonymartin — CHANGES_REQUESTED** [review]

File: packages/app/pages/lockPop.tsx [hunk 3]
```diff
@@ -0,0 +1,251 @@
+import Sidebar from '../containers/Grants/Sidebar/Sidebar';
```
Comment (anthonymartin) [inline packages/app/pages/lockPop.tsx:1]: there are a lot of unused variables on this page - is it possible to clean that up?

File: packages/app/pages/lockPop.tsx [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+    const [errorModal, setErrorModal] = useState<string>('invisible');
+    const [errorMsg, setErrorMsg] = useState<string>('');
+
+    const stakingAddress = '0xa513E6E4b8f2a923D98304ec87F64353C4D5C853';
+    const mockERCAddress = '0x5FC8d32690cc91D4c39d9d3abcBD16989F875707';
+    const timeToSeconds = { '1 week': 604800, '1 month': 2635200, '3 months': 7905600, '6 months': 15811200, '1 year': 31622400, '4 years': 126489600 };
+
+    function submitVotes() {}
+
+    async function lockPop(amountToLock, amountOfTime = 606666) {
```
Comment (anthonymartin) [inline packages/app/pages/lockPop.tsx:52]: amountToLock is not used here

**Comment by anthonymartin** [comment]
@Amirjab21 nice overall job. I've made some tweaks and polished up a few things. There were somethings that weren't functioning at all (like staking after approving an amount). Should be all good now, so I'll merge

**Pull request merged**